docs/getstarted/core_concepts.md

```diff
@@ -89,8 +84,9 @@ different datasets. When using FloydHub, datasets are always kept separate from
 code.
 
 ### Why keep datasets separate from code?
-A data scientist tweaks his/her code often during the process of creating a
-deep-learning model. However, he/she doesn't change the underlying data nearly
+
+A data scientist you tweak your code often during the process of creating a
```

Comment:
Ah, this should be "As a data scientist ..."

Reply:
Good catch